wafer_space/shuttles/static/shuttles/js/assignment_dashboard.js

```
@@ -24,17 +24,307 @@
+      renderAutocompleteResults(currentSlotSize);
+      // Scroll active item into view
+      var activeItem = resultsDiv.querySelector('.autocomplete-item.active');
+      if (activeItem) {
+        activeItem.scrollIntoView({ block: 'nearest', behavior: 'smooth' });
+      }
+    } else if (event.key === 'ArrowUp') {
+      event.preventDefault();
+      activeIndex = Math.max(activeIndex - 1, 0);
+      renderAutocompleteResults(currentSlotSize);
+      // Scroll active item into view
+      var activeItem = resultsDiv.querySelector('.autocomplete-item.active');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'activeItem'. Consider to delete it or rename it.

'activeItem' is defined here:


(lint/suspicious/noRedeclare)